content/commands/flushdb/index.md

```diff
@@ -66,6 +66,7 @@ It is possible to use one of the following modifiers to dictate the flushing mod
 
 * An asynchronous `FLUSHDB` command only deletes keys that were present at the time the command was invoked. Keys created during an asynchronous flush will be unaffected.
 * This command does not delete functions.
+* When using Redis Cluster, this command is identical to `FLUSHALL` since a Redis Cluster supports only database zero.
```

Comment:
Do we call it "database zero"? If not then maybe "only one database with an ID of zero", or something?

Reply:
Thank you @andy-stark-redis . That one spun my head too, and I've modified it per your suggestion.